src/views/v2/list/event/description.php

```diff
@@ -15,6 +15,6 @@
 $event    = $this->get( 'event' );
 $event_id = $event->ID;
 ?>
-<div class="tribe-events-calendar-list__event-description tribe-common-b1">
+<div class="tribe-events-calendar-list__event-description tribe-common-b2">
```

Comment:
I'm still seeing `b1` here, there is one event in list view that has `b2` styles it seems, but all other events have `b1` styles. Check with matt/rachel?

Reply:
Unfortunately Matt is not here, and he was in charge of this refresh. I recall he mentioned that updating the size for this was one of the changes, but maybe he referred to the featured only. Asked Rachel and she's not sure.

Reply:
I'm going off the latest designs on invision. Confirming with Matt/Rachel is best.

Reply:
confirmed with design, this is good.